src/_flow/samlper-thrift.js

```diff
@@ -81,15 +27,24 @@ declare type RateLimitingSamplingStrategy = {
     maxTracesPerSecond: number
 };
 
+// OperationSamplingStrategy defines a sampling strategy for a given operation
+// that randomly samples a fixed percentage of traces.
+declare type OperationSamplingStrategy = {
+    operation: string,
+    probabilisticSampling: ProbabilisticSamplingStrategy
+};
+
+// PerOperationSamplingStrategies defines a collection of sampling strategies
+// per operation name, and a pair of parameters for the default sampling strategy
+// applicable to unknown operation names.
+declare type PerOperationSamplingStrategies = {
+    defaultSamplingProbability: number,
+    defaultLowerBoundTracesPerSecond: number,
+    perOperationStrategies: Array<OperationSamplingStrategy>
+}
+
 declare type SamplingStrategyResponse = {
```

Comment:
PerOperationSamplingStrategies needs to be added here. I guess that'll be part of a separate PR?